lib/Doctrine/Migrations/DependencyFactory.php

```diff
@@ -151,11 +213,11 @@ public function getSchemaDumper() : SchemaDumper
     private function getSchemaProvider() : SchemaProvider
     {
         return $this->getDependency(SchemaProvider::class, function () : SchemaProvider {
-            if ($this->em === null) {
-                throw new MissingDependency('The doctrine entity manager should be provided in order to be able to instantiate SchemaProvider');
+            if (! $this->hasEntityManager()) {
```

Comment:
This check is superfluous since the `getEntityManager` call will throw the same exception if it can't find it.